README.md

```diff
@@ -48,14 +48,14 @@ Here the integration of a Timer is shown with a simple Arduino Sketch toggling t
   };
   ```
 
-* setup: set LED pin to output; create recurring Timer, inject specific TimerAdapter
+* setup: set LED pin to output; create recurring Timer, inject specific SpinTimerAction
 
   ```c++
   //The setup function is called once at startup of the sketch
   void setup()
   {
     pinMode(LED_BUILTIN, OUTPUT);
-    new SpinTimer(new BlinkTimerAdapter(), Timer::IS_RECURRING, BLINK_TIME_MILLIS);
+    new SpinTimer(BLINK_TIME_MILLIS, new BlinkTimerAction(), Timer::IS_RECURRING);
```

Comment:
The `isRecurring `paramter shall be set to `SpinTimer::IS_RECURRING`, otherwise there will be a compiler error.
In order to get the same behavior as before, the last parameter `isAutostart `shall be set different to the default: `SpinTimer::IS_AUTOSTART`.

**The correct line shall read as:**
`new SpinTimer(BLINK_TIME_MILLIS, new BlinkTimerAction(), SpinTimer::IS_RECURRING, SpinTimer::IS_AUTOSTART);`